PrebidMobile/PrebidServerAdapter/src/main/java/org/prebid/mobile/prebidserver/PrebidServerAdapter.java

```diff
@@ -44,11 +44,41 @@
     public void requestBid(final Context context, final BidManager.BidResponseListener bidResponseListener, final ArrayList<AdUnit> adUnits) {
         this.adUnits = adUnits;
         this.weakReferenceLisenter = new WeakReference<BidManager.BidResponseListener>(bidResponseListener);
-        JSONObject postData = getPostData(context, adUnits);
-        new ServerConnector(postData, this, getHost(), context).execute();
+        // Batch 10 calls for each request to server
+
+        ArrayList<ArrayList<AdUnit>> adUnitsList = getAdUnitLists(adUnits);
+        if (adUnitsList != null && !adUnitsList.isEmpty()) {
+            for (ArrayList toBeRequested : adUnitsList) {
+                JSONObject postData = getPostData(context, toBeRequested);
+                new ServerConnector(postData, this, getHost(), context).execute();
+            }
+        }
+    }
+
+    ArrayList<ArrayList<AdUnit>> getAdUnitLists(ArrayList<AdUnit> adUnits) {
+        ArrayList<ArrayList<AdUnit>> adUnitsList = new ArrayList<>();
+        int len = adUnits.size();
+        int i = len / 10;
```

Comment:
can you make this a constant that can be modified in future? In current code, if i need to modify from 10 to say 15, i need to modify this in 4 different places...